src/main/resources/templates/checkout.html

```
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="utf-8" />
+  <link rel="stylesheet" type="text/css" href="/css/style.css" />
+  <script src="https://js.tosspayments.com/v2/standard"></script>
+</head>
```

HTMLHint (1.9.2)

[error] <title> must be present in <head> tag.

(title-require)